src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -164,13 +201,22 @@ public GraphNode protoNodeToGraphNode(Node thisNode) {
 
   /*
    * Changes the graph according to the given mutation object
+   *
    * @param mut the mutation to affect
+   *
```

Comment:
I also think the formatter is adding the newlines between the @params. I deleted them and they're back after I pulled toggle with the formatter's changes. I don't think it's recognizing the comment as javadoc. I can fix this in the refactor PR because the functions move files if that's okay :)

Reply:
Ah, I think it's because the comment is not JavaDoc ;) 
JavaDoc comments start with `/**`, not `/*`. Comments that start with `/*` are simply multiline comments. 
Fixing this in the refactor PR also sgtm!